crates/turbopack-node/src/pool.rs

```diff
@@ -55,11 +60,61 @@ impl Drop for RunningNodeJsPoolProcess {
 
 const CONNECT_TIMEOUT: Duration = Duration::from_secs(30);
 
+/// Pipes the `stream` from `final_stream`, but uses `shared` to deduplicate
+/// lines that has beem emitted by other `handle_output_stream` instances with
+/// the same `shared` before.
+async fn handle_output_stream(
+    stream: impl AsyncRead + Unpin,
+    shared: Arc<Mutex<IndexSet<(Arc<[u8]>, u32)>>>,
+    mut final_stream: impl AsyncWrite + Unpin,
+) {
+    let mut buffered = BufReader::new(stream);
+    let mut own_output = HashMap::<Arc<[u8]>, u32>::new();
+    let mut buffer = Vec::new();
+    loop {
+        match buffered.read_until(b'\n', &mut buffer).await {
+            Ok(0) => {
+                break;
+            }
+            Ok(_) => {
+                let line = Arc::from(take(&mut buffer).into_boxed_slice());
+                let occurances = match own_output.entry(Arc::clone(&line)) {
+                    Entry::Occupied(mut e) => {
+                        let c = e.get_mut();
+                        *c += 1;
+                        *c
+                    }
+                    Entry::Vacant(e) => {
+                        e.insert(0);
+                        0
+                    }
+                };
```

Comment:
```suggestion
                let occurances = own_output.entry(Arc::clone(&line))
                    .and_modify(|e| { *e += 1 })
                    .or_insert(0);
```

Shouldn't it insert `1` though?